### PR TITLE
reviewers text comments addressed

### PR DESCRIPTION
**Pull request by cclarkson**
P2 references updated

Line 156: “ this should be “L995F”

Page 14. Positive selection on non-synonymous alleles: using EHH analysis the authors found significant differences in haplotype homozygosity between different genetic backgrounds carrying resistance alleles, suggesting differences in the timing and/or strength of selection. As the Ag1000 phase 2 dataset encompasses mosquito collection spanning over a decade (2000 to 2012), could the authors comment on the potential impact of the temporal sampling scheme on the observed selection events?

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -431,7 +431,21 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
...
+%
+This temporal spread allows, albeit with low-resolution, tracking of haplotype groups through time.
+%
+The spatially widespread F1 group contains haplotypes from samples collected between 2009-2012 (Figure \ref{fig:map}, \cite{clarkson2019genome}), including within just the 'ancestral node' of the median joining network (Figure \ref{fig:networks}).
+%
+To enable the large geographic spread found, even in this central F1 node (\textit{e.g.} Ghana to Angola), suggests the F1 group existed for some time before the earliest collection in 2009.
+%
+Haplotype groups F4, F5 and S2, all carry haplotypes from samples collected in Cameroon (2009) and Gabon (2000).
+%
+With 2009 and 2000 collections found within the same nodes (Figure \ref{fig:networks}), these data demonstrate how, even in these high diversity organisms \cite{Ag1000gConsortium2017}, some nucleotide sequences can persist unchanged for almost a decade.
```
Comment (alimanfoo) [inline main.tex]:
```suggestion
%%
Populations carrying \textit{kdr} alleles were collected between the years 2009 and 2012, with the exception of Gabon, which was collected in 2000.
%
This temporal spread allows, albeit with low-resolution, tracking of haplotypes through time.
%
The spatially widespread F1 group contains haplotypes from samples collected between 2009-2012 (Figure \ref{fig:map}, \cite{clarkson2019genome}) (Figure \ref{fig:networks}).
%
We still do not know how fast insecticide resistance alleles can travel between these countries, but the large geographic spread suggests the F1 haplotype group originated some considerable time before the earliest collection in 2009.
%
Haplotype groups F4, F5 and S2, all carry haplotypes from samples collected in Cameroon (2009) and Gabon (2000).
%
These observations demonstrate that, even in mosquito populations with high levels of genetic diversity and large effective population size \cite{Ag1000gConsortium2017}, nucleotide sequences carrying alleles under strong selection can persist unchanged for almost a decade.
```

Some suggestions to tighten up.

File: main.tex [hunk 2]
```diff
@@ -515,6 +529,14 @@ \subsection*{Positive selection for resistance alleles}
 Longer shared haplotypes indicate a more recent common ancestor, and thus some of these core haplotypes may have experienced more recent and/or more intense selection than others.
 %%
 
+%%
+As sample collections took place over 12 years (2000-2012), it might be expected that core haplotypes appearing earlier in our sampling would have smaller shared haplotypes due to increased opportunity for recombination and mutation.
+%
+However, no correlation was found between the year a core haplotype was first detected and the median length (r(8)=0.03, p=0.93,  Supplementary Figure \ref{fig:pspd}).
+%
+It should be noted that, due to the retrospective nature of this study, the detection date of a core haplotype (or haplotype group) does not signify its origin.
```
Comment (alimanfoo) [inline main.tex]: This sentence isn't clear what you mean. Do you need it at all?

**Review by alimanfoo — COMMENTED** [review]
Hi Chris, couple of comments...

File: main.tex (hunk 2 above)
Comment (cclarkson) [reply main.tex]: Removed.

**Pull request merged**